fleet_notifications/state_checker.py

```
@@ -1,110 +1,152 @@
+                    # Get car data
                     try:
-                        orders[state.order_id] = order_api.get_order(car_id=car_id, order_id=state.order_id)
+                        car = self.car_api.get_car(car_id)
+                        car_id = car.id
+                        if(car.car_admin_phone is None):
+                            logger.warning(f"Car {car_id} has no admin phone number.")
+                            admin_phone = ""
+                        else:
+                            admin_phone = car.car_admin_phone.phone
+                        under_test = car.under_test
                     except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)